live-examples/js-examples/expressions-functionexpression.html

```diff
@@ -0,0 +1,14 @@
+<pre>
+<code id="static-js">try {
+  notHoisted();
+}
+catch(e) {
+  console.log(e);
+  // expected output: TypeError: notHoisted is not a function
+}
+
+var notHoisted = function() {
+  console.log("bar");
+};
```

Comment:
I feel as if this example is confusing - yes, not-hoistedness is an important aspect of function expressions, but it's not the `#1` most basic thing about them, and it's odd for the first simplest example to be one that doesn't work.

Reply:
Hmm. Yes I think you're right.